uTinyRipperCore/Parser/Classes/TrailRenderer/TrailRendererParamaters.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using uTinyRipper.AssetExporters;
+using uTinyRipper.Classes;
+using uTinyRipper.Classes.AnimationClips;
+using uTinyRipper.Classes.ParticleSystems;
+using uTinyRipper.SerializedFiles;
+using uTinyRipper.YAML;
+using Object = uTinyRipper.Classes.Object;
+
+namespace uTinyRipper.Parser.Classes.TrailRenderers
+{
+	public struct TrailRendererParameters : IAssetReadable, IYAMLExportable
+	{
+		public static int GetSerializedVersion(Version version)
+		{
+			if (version.IsGreaterEqual(2018, 3))
+			{
+				return 3;
+			}
+			if (version.IsGreaterEqual(5))
+			{
+				return 2;
+			}
+			return 1;
```

Comment:
Min version is 2
https://github.com/mafaca/UtinyRipper/blob/8a4be26ea3ea7bf72b210c34fd2e7a13a1724848/uTinyRipperCore/Parser/Classes/AnimationClip/AnimationClip.cs#L228-L229